ERROR - 2021-03-23 13:50:29 --> mysqli_sql_exception: Access denied for user ''@'localhost' to database 'ci411' in C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MySQLi\Connection.php:314
Stack trace:
#0 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MySQLi\Connection.php(314): mysqli->query('CREATE DATABASE...')
#1 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(713): CodeIgniter\Database\MySQLi\Connection->execute('CREATE DATABASE...')
#2 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE DATABASE...')
#3 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\Forge.php(222): CodeIgniter\Database\BaseConnection->query('CREATE DATABASE...')
#4 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Commands\Database\CreateDatabase.php(146): CodeIgniter\Database\Forge->createDatabase('ci411')
#5 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\CreateDatabase->run(Array)
#6 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(80): CodeIgniter\CLI\Commands->run('db:create', Array)
#7 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(58): CodeIgniter\CLI\CommandRunner->index(Array)
#8 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(932): CodeIgniter\CLI\CommandRunner->_remap('index', 'db:create')
#9 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#10 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Console.php(61): CodeIgniter\CodeIgniter->run()
#12 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\spark(66): CodeIgniter\CLI\Console->run()
#13 {main}
ERROR - 2021-03-23 14:12:55 --> ErrorException: SQLite3::exec(): table `users` already exists in C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php:146
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\Users\\bob\\Do...', 146, Array)
#1 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php(146): SQLite3->exec('CREATE TABLE `u...')
#2 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(713): CodeIgniter\Database\SQLite3\Connection->execute('CREATE TABLE `u...')
#3 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#4 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\Forge.php(525): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#5 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\app\Database\Migrations\2021-03-23-184707_Users.php(36): CodeIgniter\Database\Forge->createTable('users')
#6 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(1057): App\Database\Migrations\Users->up()
#7 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(202): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#8 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Commands\Database\Migrate.php(94): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(80): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(58): CodeIgniter\CLI\CommandRunner->index(Array)
#12 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(932): CodeIgniter\CLI\CommandRunner->_remap('index', 'migrate')
#13 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#14 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Console.php(61): CodeIgniter\CodeIgniter->run()
#16 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\spark(66): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2021-03-23 14:18:14 --> ErrorException: SQLite3::exec(): table `users` already exists in C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php:146
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\Users\\bob\\Do...', 146, Array)
#1 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php(146): SQLite3->exec('CREATE TABLE `u...')
#2 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(713): CodeIgniter\Database\SQLite3\Connection->execute('CREATE TABLE `u...')
#3 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#4 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\Forge.php(525): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#5 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\app\Database\Migrations\2021-03-23-184707_Users.php(36): CodeIgniter\Database\Forge->createTable('users')
#6 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(1057): App\Database\Migrations\Users->up()
#7 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(202): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#8 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Commands\Database\Migrate.php(94): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(80): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(58): CodeIgniter\CLI\CommandRunner->index(Array)
#12 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(932): CodeIgniter\CLI\CommandRunner->_remap('index', 'migrate', 'down')
#13 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#14 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Console.php(61): CodeIgniter\CodeIgniter->run()
#16 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\spark(66): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2021-03-23 14:18:41 --> ErrorException: SQLite3::exec(): table `users` already exists in C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php:146
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\Users\\bob\\Do...', 146, Array)
#1 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php(146): SQLite3->exec('CREATE TABLE `u...')
#2 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(713): CodeIgniter\Database\SQLite3\Connection->execute('CREATE TABLE `u...')
#3 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#4 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\Forge.php(525): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#5 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\app\Database\Migrations\2021-03-23-184707_Users.php(36): CodeIgniter\Database\Forge->createTable('users')
#6 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(1057): App\Database\Migrations\Users->up()
#7 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(202): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#8 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Commands\Database\Migrate.php(94): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(80): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(58): CodeIgniter\CLI\CommandRunner->index(Array)
#12 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(932): CodeIgniter\CLI\CommandRunner->_remap('index', 'migrate')
#13 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#14 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Console.php(61): CodeIgniter\CodeIgniter->run()
#16 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\spark(66): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2021-03-23 14:19:43 --> ErrorException: SQLite3::exec(): table `users` already exists in C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php:146
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'SQLite3::exec()...', 'C:\\Users\\bob\\Do...', 146, Array)
#1 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\SQLite3\Connection.php(146): SQLite3->exec('CREATE TABLE `u...')
#2 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(713): CodeIgniter\Database\SQLite3\Connection->execute('CREATE TABLE `u...')
#3 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\BaseConnection.php(641): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `u...')
#4 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\Forge.php(525): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `u...')
#5 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\app\Database\Migrations\2021-03-23-184707_Users.php(36): CodeIgniter\Database\Forge->createTable('users')
#6 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(1057): App\Database\Migrations\Users->up()
#7 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Database\MigrationRunner.php(202): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#8 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\Commands\Database\Migrate.php(94): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(80): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\CommandRunner.php(58): CodeIgniter\CLI\CommandRunner->index(Array)
#12 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(932): CodeIgniter\CLI\CommandRunner->_remap('index', 'migrate', 'users', 'down')
#13 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(CodeIgniter\CLI\CommandRunner))
#14 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#15 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\system\CLI\Console.php(61): CodeIgniter\CodeIgniter->run()
#16 C:\Users\bob\Documents\PHP_WebSites\codeigniter\ci411\spark(66): CodeIgniter\CLI\Console->run()
#17 {main}
